Makefile

```
@@ -82,8 +82,12 @@ $(extension_node_modules): node_modules
 $(extension_dists): src/extensions/npm/extensions/dist
 	cd $(@:/dist=) && ../../node_modules/.bin/npm run build
 
+.PHONY: clean-old-extensions
+clean-old-extensions:
+	./scripts/clean-old-extensions.sh
```

Why separate file vs calling `find` directly here?

```suggestion
	find extensions -d 1 -type d '!' -exec test -e "{}/package.json" \; -exec rm -r {} \; || exit 0
```

---

I did it because I was initially using a different mechanism but then changed it to use `find`.

Had to do some more experimentation because `||` doesn't work in `Makefile` and I was getting errors without it.